scalableminds/webknossos
# Virtualize Bounding Box List

### URL of deployed dev instance (used for testing):
- https://virtualizebb.webknossos.xyz/
- e.g. some bounding boxes are in this annotation: https://virtualizebb.webknossos.xyz/links/hDU6gJX0zhrR8jRS

### Steps to test:
- Open an annotation and go to bounding box tab.
- Make sure you have at least 25 bounding boxes. The list should scroll quite some content.
- UI wise, nothing should be different than before, test that. It should look and behave as before (layout, scrolling, etc)
- Check that only the visible bounding boxes (plus some for padding) are loaded, e.g. by inspecting the html. 

### Issues:
- fixes #7938 

------
(Please delete unneeded items, merge only when none are left open)
